### DISCUSSION TITLE
w-[38px]  The custom style of the selection number is invalid

### DISCUSSION THREAD

**Discussion by pengshengjie** [tailwindlabs/tailwindcss#10009] [Help]
Using react-app-rewired;
npm install -D tailwindcss@latest postcss@latest autoprefixer@latest
npx tailwindcss init

![image](https://user-images.githubusercontent.com/117100743/205998607-206443f3-90af-43c6-8bbd-95430cef0e41.png)

**Comment by wongjn** [comment]
Some things to check:
- The `content` file globs in the Tailwind configuration cover the file where the `w-[38px]` class appears.
- The `w-[38px]` class has [**not** been constructed with string interpolation](https://tailwindcss.com/docs/content-configuration#dynamic-class-names).
- Tailwind has been integrated into your build system correctly.
- Tailwind has run a CSS compilation.
- The `@tailwind` directives exist in CSS source code that is actually used in the project.
